mocha.opts

```diff
@@ -1,6 +0,0 @@
---require ts-node/register
---require source-map-support/register
---require src/test/helpers/mocha-bootstrap.js
---recursive
```

Comment:
Do we not need this anymore?

Reply:
> With recursive matching enabled, the following is the same as passing --recursive:
> $ mocha "./spec/**/*.js"

https://mochajs.org/#the-test-directory

hmmm... but maybe. depends on the shell? I'll leave it since it doesn't hurt.